base/applications/charmap/map.c

```diff
@@ -256,7 +256,7 @@ SetFont(PMAP infoPtr,
                          GGI_MARK_NONEXISTING_GLYPHS) != GDI_ERROR)
     {
         j = 0;
-        for (i = 0; i < MAX_GLYPHS; i++)
+        for (i = ' ' + 1; i < MAX_GLYPHS; i++)
```

Comment:
Hmm... This should be possibly investigated more. Suppose I try to view a font for which I have printable glyphs for the first characters (0x00 to 0x20); in theory I would like to see these as well?

Reply:
With such a hypothetical font, 0x20 might be a non-space printable character then? How would we tell that case apart from regular space?
If you have such a font to test with I'd be very interested to see how Windows charmap displays it. If this is purely hypothetical, then I think this solution is reasonable?

Reply:
Yes here it was more like "something to think about" / test on Windows / find a nice solution.
For the time being what has been done here doesn't block any merging.

Reply:
Fair enough, and I'm not 100% sure of the answer here.  I would like to note that the loop from 0 to ' ' actually fixed more issues with some fonts having 'carriage return' as a valid glyph.  So I do like this new loop approach better than simply skipping ' '.